src/replica/bulk_load/replica_bulk_loader.cpp

```diff
@@ -520,38 +520,25 @@ void replica_bulk_loader::download_files(const std::string &provider_name,
 
     // download sst files asynchronously
     if (!_metadata.files.empty()) {
-        const file_meta &f_meta = _metadata.files[0];
+        const file_meta f_meta = _metadata.files.back();
```

Comment:
It's needed to protect the scope by a write lock.

Reply:
Thank you for the reminder, I've learned a lot from your suggestions. I'll make the necessary modifications to ensure the code passes the unit tests.